src/util/index.js

```diff
@@ -745,6 +745,39 @@ var util = module.exports = {
     return {map: mapParams, layer: layerParams};
   },
 
+  /**
+   * Return the coordinate associated with the center of the perimeter formed
+   * from a list of points.
+   *
+   * @param {geo.geoPosition[]} coor An array of coordinates.
+   * @returns {geo.geoPosition|undefined} The position for the center, or
+   *    `undefined` if no such position exists.
+   */
+  centerFromPerimeter: function (coor) {
```

Comment:
@manthey it would be good to describe the math in words to explain what is done in this snippet.

Reply:
@aashish24 I added a bit more to the function comment.

Reply:
thanks, I found it useful.